cli/src/commands/split.rs

```diff
@@ -228,16 +298,15 @@ pub(crate) fn cmd_split(
 
     let legacy_bookmark_behavior = tx.settings().get_bool("split.legacy-bookmark-behavior")?;
     if legacy_bookmark_behavior {
-        // Mark the commit being split as rewritten to the second commit. This
-        // moves any bookmarks pointing to the target commit to the second
-        // commit.
```

Comment:
Why did you remove this comment? Is it no longer valid?

Reply:
At first I completely removed the legacy bookmark config and used this call to change the parent in the rewrite, so yes it was no longer valid. (I'm not saying that necessarily the best way to do it :))

Since I reintroduced the legacy bookmark config and had to set the parent explicitly in the rewrite, I think that I can completely remove the call associated with that commit.

Reply:
I confused the situation with legacy change id / legacy bookmark.
We still need the call, and the comment that goes with it